src/Composer/Command/AuditCommand.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Composer\Command;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Composer\Package\PackageInterface;
+use Composer\Repository\InstalledRepository;
+use Composer\Repository\RepositoryInterface;
+use Composer\Util\Auditor;
+use Symfony\Component\Console\Input\InputOption;
+
+class AuditCommand extends BaseCommand
+{
+    protected function configure(): void
+    {
+        $this
+            ->setName('audit')
+            ->setDescription('Checks for security vulnerability advisories for installed packages.')
+            ->setDefinition(array(
+                new InputOption('no-dev', null, InputOption::VALUE_NONE, 'Disables auditing of require-dev packages.'),
+                new InputOption('format', 'f', InputOption::VALUE_OPTIONAL, 'Output format. Must be "table", "plain", or "summary".', Auditor::FORMAT_TABLE),
+                new InputOption('locked', null, InputOption::VALUE_NONE, 'Audit based on the lock file instead of the installed packages.'),
+            ))
+            ->setHelp(
+                <<<EOT
+The <info>audit</info> command checks for security vulnerability advisories for installed packages.
+
+If you do not want to include dev dependencies in the audit you can omit them with --no-dev
+
+Read more at https://getcomposer.org/doc/03-cli.md#audit
+EOT
+            )
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $composer = $this->requireComposer($input->getOption('no-plugins'), $input->getOption('no-scripts'));
+        $packages = $this->getPackages($input);
+        $httpDownloader = $composer->getLoop()->getHttpDownloader();
+
+        if (count($packages) === 0) {
+            $this->getIO()->writeError('No packages - skipping audit.');
+            return 0;
+        }
+
+        $auditor = new Auditor($httpDownloader);
+        return $auditor->audit($this->getIO(), $packages, $input->getOption('format'), false);
+    }
+
+    /**
+     * @param InputInterface $input
+     * @return PackageInterface[]
+     */
+    private function getPackages(InputInterface $input): array
+    {
+        $composer = $this->requireComposer($input->getOption('no-plugins'), $input->getOption('no-scripts'));
```

Comment:
No need to forward those they are used as default value already for conciseness

```suggestion
        $composer = $this->requireComposer();
```